src/styles/custom.less

```diff
@@ -12,6 +13,25 @@
 @pink:   #ff5b77;
 @purple: #4757b2;
 
+@primary-color: @purple;
+
 @font-family-sans-serif: "Whitney", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
 @font-family-monospace:  "WhitneyBold", Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
 @font-family-base:       @font-family-sans-serif;
+
+@body-background: @white-darker;
+@component-background: @white-darker;
+@font-size-base: 14px;
```

Comment:
Font-size should be 16px by default, we dont use 14px font-size on the new-design.